tools/e2etesting/TestEventProcessor/TestEventProcessor.Businesslogic/TelemetryValidator.cs

```diff
@@ -289,6 +289,12 @@ private Task Client_ProcessEventAsync(ProcessEventArgs arg)
                     continue;
                 }
 
+                // OPC PLC contains bad fast and slow nodes that drop messages by design.
+                // We will ignore entries that do not have a value.
+                if (entryValue is null) {
```

Comment:
If we do not subscribe to bad nodes, that this should not be necessary right?

Reply:
+1

Reply:
Well, yes, but that's if we assume that no-one will accidentally include them in a subscription again. And I would rather make sure that test event processor can handle those situations instead of hoping for that. So I will keep this check here.

Reply:
ok, the first one that will really test BadNodeIds should add an setting to the configuration